assets/src/edit-story/components/canvas/addpage/plus.svg

```diff
@@ -1,3 +1,3 @@
-<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M16 0V16M16 32V16M16 16H0M16 16H32" stroke="currentColor" stroke-width="6"/>
+<svg width="16" height="16" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I actually think it's better to remove `width` and `height` and style it in CSS (`viewbox` will then determine relative sizes).